packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -1008,6 +1008,7 @@ class ListView extends BoxScrollView {
   /// child that could possibly be displayed in the list view instead of just
   /// those children that are actually visible.
   ///
+  /// Note that this constructor does not dynamically change the [List] once generated.
```

Comment:
Also, it is unclear to me what you actually mean by this comment. Like any widget you'll have to rebuild the widget if you want to change any properties.

Reply:
Sorry for late reply. I tried to modify docs as you said. Hope this one would be better